### PR TITLE
Bump WP requirement in plugin

### LINKED ISSUE #7592: CI: Run tests against WP 5.5
## Task Description

As part of #7551, we should change the e2e tests GitHub action so that tests are run against WP latest and 5.5.x.

https://github.com/google/web-stories-wp/blob/c4b2f02b98713b17962f283a5f2205f93d56fbce/.github/workflows/tests-e2e.yml#L45-L47

The same goes for the PHP unit tests, where the PHP 5.6 & WP 5.3 should be changed to run against WP 5.5.

https://github.com/google/web-stories-wp/blob/c4b2f02b98713b17962f283a5f2205f93d56fbce/.github/workflows/tests-unit-php.yml#L56-L57

### LINKED ISSUE #7598: Bump WP requirement in plugin
## Task Description

Need to bump the WP requirement to 5.5 in at least these places:

https://github.com/google/web-stories-wp/blob/c4b2f02b98713b17962f283a5f2205f93d56fbce/readme.txt#L5

https://github.com/google/web-stories-wp/blob/c4b2f02b98713b17962f283a5f2205f93d56fbce/web-stories.php#L16

https://github.com/google/web-stories-wp/blob/c4b2f02b98713b17962f283a5f2205f93d56fbce/web-stories.php#L51

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Update min required WP version. Change min version and test at some time. 

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7598
Fixes #7592

### CODE REVIEW

**Comment by miina** [comment]
E2E tests are failing with the following message:
`Error: No plugins installed.`

Thoughts?

Edit: tried re-running, same result.

**Comment by spacedmonkey** [comment]
@miina I think I found the issue. As of WordPress 5.5, there new header when define a plugin. This defines the min version of WordPress required. Gutenberg required 5.6 and we are installing it on 5.5. It errors out correctly. I have put a fix in.

File: bin/local-env/install-wordpress.sh [hunk 1]
```diff
@@ -129,12 +129,16 @@ echo -e $(status_message "Activating Web Stories plugin...")
 wp plugin activate web-stories --quiet
 
 # Install & activate testing plugins.
+
+# Only install gutenberg on latest version of WordPress. 
+if [ "$WP_VERSION" == "latest" ]; then
```
Comment (miina) [inline bin/local-env/install-wordpress.sh:134]: You mentioned in the comment that Gutenberg requires 5.6 -- should this specify installing on any version starting from 5.6?

Comment (miina) [reply bin/local-env/install-wordpress.sh:134]: Or since we're testing only 5.5 and the latest it doesn't matter, I suppose?

**Review by miina — APPROVED** [review]

File: bin/local-env/install-wordpress.sh (hunk 1 above)
Comment (spacedmonkey) [reply bin/local-env/install-wordpress.sh:134]: The most careful thing is only install on latest. We can always change later.

**Pull request merged**